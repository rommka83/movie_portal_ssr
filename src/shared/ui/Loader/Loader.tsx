import React, { FC, HTMLAttributes } from 'react';
import styles from './loader.module.css';
import classNames from 'classnames';

export const Loader: FC<HTMLAttributes<HTMLDivElement>> = ({ className }) => {
  return (
    <div className={classNames(styles.ldsSpinner, className)}>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
    </div>
  );
};
