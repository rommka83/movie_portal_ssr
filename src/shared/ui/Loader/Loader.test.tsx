import renderer from 'react-test-renderer';
import { Loader } from './Loader';

describe('проверка Loader', () => {
  test('отрисовка Loader', () => {
    const component = renderer.create(<Loader />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
