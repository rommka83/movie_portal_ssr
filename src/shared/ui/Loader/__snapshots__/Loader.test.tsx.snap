// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`проверка Loader отрисовка Loader 1`] = `
<div
  className="ldsSpinner"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
